Fix keycloak export sql for postgres (#2099)
FusionAuth/fusionauth-site: site/docs/v1/tech/migration-guide/keycloak.adoc

> Here's a SQL script that will query the needed fields. Update `RealmID` with the name of the Keycloak realm to export, gathered from the <<Obtaining User Data>> step.

Reviewer: ```suggestion
Here are SQL scripts that will query the needed fields. Update `RealmID` with the name of the Keycloak realm to export, gathered from the <<Obtaining User Data>> step. The scripts are different depending on if you are exporting from MySQL or PostgreSQL.
```

> 

Reviewer: ```suggestion

=== Exporting From MySQL
```

Reviewer: And then add both of these sections to the table of contents at the top of the file.

cat keycloak-export.sql | mysql -u USER -p keycloak| sed 's/\t/,/g' > out.csv
----

> 

Reviewer: ```suggestion
=== Exporting From PostgreSQL

```
